git/clone/rwx-package.yml

```diff
@@ -121,24 +121,7 @@ tasks:
 
   - key: get-latest-sha-for-ref
     use: setup
-    run: |
-      if [[ "${CHECKOUT_REF}" == "HEAD" ]]; then
-        echo "Error: Cloning from HEAD is ambiguous. If you want to clone from the HEAD of a particular branch" >&2
-        echo "pass in the name of that branch as the ref instead." >&2
-        exit 1
-      fi
-
-      CREDENTIAL_ARG=""
-      if [[ "${CREDENTIAL_HELPER}" != "" ]]; then
-        CREDENTIAL_ARG="-c credential.helper='${CREDENTIAL_HELPER}'"
-      fi
-      COMMAND="git $CREDENTIAL_ARG ls-remote \"${CHECKOUT_REPOSITORY}\" \"${CHECKOUT_REF}\""
-      RESOLVED_SHA=$(eval $COMMAND | awk '{print $1}')
-      if [[ $RESOLVED_SHA == "" ]]; then
-        RESOLVED_SHA="${CHECKOUT_REF}"
-      fi
-      echo "Latest SHA for ${CHECKOUT_REF}: ${RESOLVED_SHA}"
-      printf "${RESOLVED_SHA}" >> "$MINT_VALUES/sha"
+    run: $MINT_LEAF_PATH/bin/get-latest-sha-for-ref
```

Comment:
if you have to make any other changes, `$MINT_LEAF_PATH` is now `$RWX_PACKAGE_PATH`